pkg/controllermanager/controller/shoot/shoot_control_delete.go

```diff
@@ -92,23 +92,17 @@ func (c *defaultControl) deleteShoot(o *operation.Operation) *gardenv1beta1.Last
 	// We also check whether the kube-apiserver pod exists in the Shoot namespace within the Seed cluster. If it does not,
```

Comment:
```suggestion
	// We also check whether the gardener-shoot-deletion-checkpoint ConfigMap exists in the Shoot namespace within the Seed cluster. If it does exist,
```
You could use `DeletionCheckpoint` instead of `gardener-shoot-deletion-checkpoint`. I let you to decide which one is better.